tests/components/group/test_event.py

```diff
@@ -0,0 +1,139 @@
+"""The tests for the group event platform."""
+
+from pytest_unordered import unordered
+
+from homeassistant.components.event import DOMAIN as EVENT_DOMAIN
+from homeassistant.components.event.const import ATTR_EVENT_TYPE, ATTR_EVENT_TYPES
+from homeassistant.components.group import DOMAIN
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_ENTITY_ID,
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.setup import async_setup_component
+
+
+async def test_default_state(hass: HomeAssistant) -> None:
+    """Test event group default state."""
+    await async_setup_component(
+        hass,
+        EVENT_DOMAIN,
+        {
+            EVENT_DOMAIN: {
+                "platform": DOMAIN,
+                "entities": ["event.button_1", "event.button_2"],
+                "name": "Remote control",
+                "unique_id": "unique_identifier",
+            }
+        },
+    )
+    await hass.async_block_till_done()
+    await hass.async_start()
+    await hass.async_block_till_done()
+
+    state = hass.states.get("event.remote_control")
+    assert state is not None
+    assert state.state == STATE_UNAVAILABLE
+
+    hass.states.async_set(
+        "event.button_1",
+        "on",
```

Comment:
I don't understand this. The event entity has a timestamp as state, right?

Reply:
Uh right 👍